cockpit/package.json

```diff
@@ -10,20 +10,29 @@
     "build": "webpack"
   },
   "devDependencies": {
+    "babel-cli": "^6.24.1",
     "babel-core": "^6.25.0",
     "babel-loader": "^7.0.0",
+    "babel-plugin-transform-flow-strip-types": "^6.22.0",
     "babel-plugin-transform-react-jsx": "^6.24.1",
     "babel-preset-env": "^1.5.2",
+    "babel-preset-flow": "^6.23.0",
+    "babel-register": "^6.24.1",
     "compression-webpack-plugin": "~1.0.0",
     "copy-webpack-plugin": "~3.0.1",
     "eslint": "^3.0.0",
     "eslint-loader": "~1.6.1",
     "eslint-plugin-react": "~6.9.0",
+    "flow-bin": "^0.52.0",
     "jshint": "~2.9.1",
     "jshint-loader": "~0.8.3",
-    "webpack": "^2.6.1"
+    "webpack": "^3.5.1"
   },
   "dependencies": {
-    "react-lite": "0.15.30"
+    "immutable": "^3.8.1",
```

Comment:
Let's not pull in all these dependencies just yet (instead, let's pull them in as we're starting to use them). Also, they should probably be added as dev dependencies.